Rmarkdown/Multiple_Ascending_Dose_PD_continuous.Rmd

```diff
@@ -1,314 +1,311 @@
+
+pd_data_rep_by_trt <- list()
+for(id in unique(pd_data$ID)){
+  indiv_data <- pd_data %>% subset(ID == id)
+  itrtact = unique(indiv_data$TRTACT_low2high)
+  
+  pd_data_rep_by_trt[[as.character(id)]] <- pd_data %>% 
+        subset(TRTACT_low2high == itrtact) %>% 
+        mutate(ID_rep_by_trt = ID, ID = id)
+}
+pd_data_rep_by_trt <- bind_rows(pd_data_rep_by_trt)
+
+gg <- ggplot(mapping = aes(x = TIME/24, y = LIDV)) 
+gg <- gg + geom_line(data = pd_data_rep_by_trt, 
+                     aes(group = ID_rep_by_trt), 
+                     size = 1, color = rgb(0.5,0.5,0.5), alpha = 0.3)
+gg <- gg + geom_line(data = pd_data, 
+                     aes(group = ID), size = 1)
+gg <- gg + xgx_scale_y_log10()
+gg <- gg + xgx_scale_x_time_units(units_dataset = time_units_dataset, 
+                                  units_plot    = time_units_plot)
+gg <- gg + scale_x_continuous(breaks = seq(0,max(pd_data$TIME/24)+1,7))
+gg <- gg + labs(y = pd_label)
+gg <- gg + theme(legend.position="none")
+gg + facet_wrap(~ID + TRTACT_low2high, 
+                ncol = 10)
+gg <- gg + xgx_annotate_status(status, fontsize = 4, color=rgb(0.5,0.5,1))
+
+
+```
+
+## Explore covariate effects on PD
+
+Stratify by covariates of interest to explore whether any key covariates impact response. For examples of plots and code startifying by covariate, see [Single Ascending Dose Covariate Section](Single_Ascending_Dose_PK.html#explore_covariate_effects_on_pk)
+
+**Warning** Be careful of interpreting covariate effects on PD. Covariate effects on PD could be the result of covariate effects on PK transfering to PD through the PK/PD relationship.
+
+## Explore Dose-Response Relationship
+
+One of the key questions when looking at PD markers is to determine if there is a dose-response relationship, and if there is, what dose is necessary to achieve the desired effect? Simple dose-response plots can give insight into these questions.
+
+
+### PD marker by Dose, for endpoint of interest, mean (95% CI) by Dose
+
+Plot PD marker against dose. Using summary statistics can be helpful, e.g. Mean +/- SE, or median, 5th & 95th percentiles. 
+
+Here are some questions to ask yourself when looking at Dose-Response plots: Do you see any relationship? Does response increase (decrease) with increasing dose? Are you able to detect a plateau or emax (emin) on the effect? If so, around what dose does this occur?
+
+**Warning:** Even if you don't see an Emax, that doesn't mean there isn't one. Be very careful about using linear models for Dose-Response relationships. Extrapolation outside of the observed dose range could indicate a higher dose is always better (even if it isn't).
+
+
+```{r, cache = TRUE, fig.width = 8, fig.height = 4, warning = FALSE, message= FALSE} 
+pd_data$DAY_label <- paste("Day", pd_data$PROFDAY)
+pd_data$DAY_label[pd_data$DAY_label=="Day 0"] = "Baseline"
+pd_data_to_plot <- pd_data[pd_data$DAY_label%in%c("Baseline","Day 6"),]

```

Comment:
suggest to use %>%, e.g. as follows:
pd_data_to_plot <- pd_data %>% subset(DAY_label %in% c("Baseline", "Day 6")

Reply:
ok, all changed